ykrt/src/trace/swt/asm.rs

```diff
@@ -0,0 +1,111 @@
+use capstone::prelude::*;
```

Comment:
Is everything in this file only used for testing?

Reply:
That needs documenting with `//!` at the top of this module.

Reply:
will add